.github/workflows/ci-e2e.yml

```diff
@@ -13,7 +13,7 @@ concurrency:
 jobs:
     build:
         name: Docker image build
-        runs-on: ubuntu-latest
```

Comment:
Not sure what's happening here but this diff is not expected. Here's the content from `origin/master` 🤔